budget-squirrel/src/App.js

```diff
@@ -14,6 +14,8 @@ function App() {
         <Layout>
             <Route exact={true} path={RouteConstants.ROOT} component={Splash} />
             <Route path={RouteConstants.REGISTER} component={Register} />
+            <Route path={RouteConstants.LOGIN} component={Login} />
+            <Route component={Splash} />
```

Comment:
This is why the splash page is stacked under your login page. We already have a route for the splash page. This Route is saying that for all routes, we should  render the splash page.

Reply:
I can remove this but I was hoping to address any 404 by returning them to the splash page. I was following this [link](https://codeburst.io/getting-started-with-react-router-5c978f70df91).

Reply:
Let's figure the 404 out in another PR then. So just have your login route here.